spring-boot-docs/src/main/asciidoc/using-spring-boot.adoc

```diff
@@ -892,6 +892,9 @@ devtools support, simply add the module dependency to your build:
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-devtools</artifactId>
 			<optional>true</optional>
+			<configuration>
+                    		<fork>true</fork>
+                        </configuration>
 		</dependency>
```

Comment:
I think that both changes are not good. Do you want to rework your PR (just push force on your branch to update it)? Otherwise, I can have a look to it myself.